scripts/jenkins/cloud/ansible/roles/list_packages/defaults/main.yml

```diff
@@ -20,4 +20,12 @@ state1: "after_deploy"
 state2: "after_update"
 effective_user: "{{ (cloud_product == 'ardana') | ternary('ardana','root') }}"
 local_tmp_dir: "/tmp"
-diff_tmp_dir: "{{ lookup('env', 'WORKSPACE') | ternary (workspace_path, local_tmp_dir) }}" 
+diff_tmp_dir: "{{ lookup('env', 'WORKSPACE') | ternary (workspace_path, local_tmp_dir) }}"
+build_id: "{{ lookup('env', 'BUILD_ID') if lookup('env', 'BUILD_ID') else '764' }}"
```

Comment:
what's the reason for 764?

I think the same can reached using `{{ lookup('env', 'BUILD_ID') | default('764', true) }}`

Reply:
784 value here is for manual deployment... just thinking if I should keep it there or not.

Reply:
I reworked it. So now if not defined it uses empty default and it means no diffs of old runs in manual deployment (If you wanna use it just define number in vars)

Reply:
perfect, thank you!